### ISSUE TITLE
Redesign of POI icons on map

### ISSUE THREAD

**Issue by zintor** [navit-gps/navit#733]
As there are a bunch of map POI icons from different sources with various styles and without an cooperate design, it would be nice to have a POI Icon package for Navit.

So I used the package of https://github.com/gmgeo/osmic to create some variations of icons:
![fuel](https://user-images.githubusercontent.com/22443449/51445269-87aed480-1d03-11e9-996a-43cfdc75bc7f.png)
I know, that it is a lot of work, but we could start with this package of osmic to get the most relevant icons.

The menu use the same icons. So there are some more work.

Some Screenshotd in SFOS 3 Emulator with a "SFOS"-style 96x96px:
![fuel_marker_pin1_96_96](https://user-images.githubusercontent.com/22443449/51445279-95fcf080-1d03-11e9-89d6-44ff26d9cfc8.PNG)
![fuel_marker_pin1_96_96_3d](https://user-images.githubusercontent.com/22443449/51445280-95fcf080-1d03-11e9-9773-00d7ed98cbc5.PNG)
![fuel_marker_pin2_96_96](https://user-images.githubusercontent.com/22443449/51445281-95fcf080-1d03-11e9-8cac-9fd25993e148.PNG)
![fuel_marker_pin2_96_96_3d](https://user-images.githubusercontent.com/22443449/51445282-96958700-1d03-11e9-91fa-c84c1564cded.PNG)
![fuel_marker_rect_96_96_sfos_2d](https://user-images.githubusercontent.com/22443449/51445283-96958700-1d03-11e9-9f29-33b07c4852c1.PNG)
![fuel_marker_rect_96_96_sfos_3d](https://user-images.githubusercontent.com/22443449/51445284-96958700-1d03-11e9-9e02-a829d581d737.PNG)

**Comment by mvglasow** [comment]
Looks great and is indeed much needed. While we’re at it, we might also want to think of a redesign of the default OSD layout; my suggestion would be this one: https://wiki.navit-project.org/index.php/File:Screenshot_Navit_20180831-141920.png

**Comment by hoehnp** [comment]
I am also currently working on a redesign. This should then include an auxmap with a simplified layout to avoid a map too much with POIs. However, I would guess this code still take a while as it also requires some changes to the code. What do you think?

**Comment by zintor** [comment]
Well, the layout is not my Intention to bring forward. The layout section on the [wiki](https://wiki.navit-project.org/index.php/Layout) shows good examples. Maybe the Mapnik layout could be the stanard layout.
Here I would like to discuss/get some devisions for the POI icons.
Questions are:
osmic Icons are under the CC0 1.0 Universal License. Is that ok for shipped navit?
Which shape of POI icons is the best? Pin1, Pin2, the recangled one, or … ?
Two-colored icons? Background-color and icon-outline-color? -> IMHO this could be the best to get other color combination by an search and replace script?

**Comment by mvglasow** [comment]
I was in fact referring to OSD, not map layout, but I get your message :-) I will take care of that.

CC-0 should be OK, under that license the author essentially agrees to have all their rights waived to the extent permitted under law. Should be compatible with the GPL2.

As for icon shape, I prefer the rectangle. Most icons were likely designed for a rectangular bounding box, and for any other pin shape you might end up having to scale many icons down, hampering legibility.

As for color, I like the approach OSM have taken here: assign a color to each POI category. I would folow that approach for the pin color, and keep the icon itself in white. We might want to look at night layouts separately, maybe use black icons and invert the luminance of the background color in night mode.

**Comment by metalstrolch** [comment]
One thing to remember is, that currently poi icons are as well used within the onternal gui menus. So any shape exceot rectangular looks strange. And one should remember the all black background there. So contrast color outlines are to be considered. Or a second in-osd set of icon needs to be prepared.

**Comment by zintor** [comment]
@mvglasow: which color do you mean? from [here](https://wiki.openstreetmap.org/wiki/Map_Features)?
@metalstrolch: yes, you are right. Maybe a re-structure of the icons-locations (for internal gui , nav, country, gui) could be approached.

**Comment by mvglasow** [comment]
Yes, similar to the color for map icons here. E.g. ochre for anything that provides food and drinks, brown for general amenities, blue for transport infrastructure, dark red for healthcare, purple for retail, black for religious, green for leisure/recreation… It would in fact be good to stay close to the icon colors of the OSM Carto style—it’s mature and people are likely familiar with it already.
